### PR TITLE
Update language

### PR DESCRIPTION
**Pull request by caniszczyk**
Signed-off-by: Chris Aniszczyk <caniszczyk@gmail.com>

### CODE REVIEW

**Comment by OBWANDO** [comment]
Makes sense, need to show proper commercial use of trademark

**Review by OBWANDO — APPROVED** [review]
Makes sense, need to show proper commercial use of trademark

**Review by AMZN-Liv — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Open 3D Engine
+# O3DE (Open 3D Engine)
 
-Open 3D Engine (O3DE) is an open-source, real-time, multi-platform 3D engine that enables developers and content creators to build AAA games, cinema-quality 3D worlds, and high-fidelity simulations without any fees or commercial obligations.
+O3DE (Open 3D Engine)is an open-source, real-time, multi-platform 3D engine that enables developers and content creators to build AAA games, cinema-quality 3D worlds, and high-fidelity simulations without any fees or commercial obligations.
```
Comment (Kadino) [inline README.md:3]: Missing a space between parens and is ```Engine)is```

Comment (caniszczyk) [reply README.md]: sigh thanks

**Review by Kadino — APPROVED** [review]

**Pull request merged**